pandas/_libs/tslibs/period.pyx

```diff
@@ -2404,6 +2406,13 @@ class Period(_Period):
                 value = str(value)
             value = value.upper()
             dt, reso = parse_time_string(value, freq)
+            try:
+                ts = Timestamp(value, freq=freq)
```

Comment:
don't pass freq

Reply:
A new PR is ready (waiting for the question about `else`)